resources/js/Pages/VacationRequest/Create.vue

```diff
@@ -296,8 +296,9 @@
               </InertiaLink>
               <button
                 type="submit"
-                :disabled="form.processing"
                 class="inline-flex justify-center py-2 px-4 text-sm font-medium text-white bg-blumilk-600 hover:bg-blumilk-700 rounded-md border border-transparent focus:outline-none focus:ring-2 focus:ring-blumilk-500 focus:ring-offset-2 shadow-sm"
+                :class="{ 'disabled:opacity-60': form.processing || !form.isDirty }"
+                :disabled="form.processing || !form.isDirty"
```

Comment:
Only on this page the button is not disabled, because of default state. For a while we have null in vacation type, and after that we set first vacation type (depends on user) and in this moment fronted thinks that something change, but it isn't. So, in the line (I mean after `vacationTypes.value = res.data`) [459](https://github.com/blumilksoftware/toby/pull/164/commits/d36ef71dd923bd088e6d5a3c86575f6130e6cba4#diff-e2fa9345d2a5f1749faabb2fc0bd4981c2efc3c46a6684c25cc16547a406198fR459) write `form.defaults('vacationType',vacationTypes.value[0])`.

Reply:
Now it should work properly.